kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -1104,11 +1175,14 @@ class ResidualBasedNewtonRaphsonStrategy
      * @param MoveMesh The flag that allows to move the mesh
      */
 
-    virtual void UpdateDatabase(
-        TSystemMatrixType& rA,
-        TSystemVectorType& rDx,
-        TSystemVectorType& rb,
-        const bool MoveMesh)
+    virtual void
+    UpdateDatabase(
+            TSystemMatrixType &rA,
+            TSystemVectorType &rDx,
+            TSystemVectorType &rb,
+            const bool MoveMesh,
+            const bool complete_update = false
```

Comment:
this flag `complete_update` is not used in this function. But for example in the line

https://github.com/KratosMultiphysics/Kratos/blob/2095e9d21bbba1017ff431fa72f0864940ed992a/kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h#

the function is used with the flag set true, which should not have any effect. Is this something which is planned to be used in the future ?

Reply:
It is used in the line_search strategy which is derived

Reply:
Ok .. was just checking if you missed something here !